integrations/express/middleware.ts

```diff
@@ -160,28 +160,30 @@ export function afterMiddleware(keploy: Keploy, req: Request, res: Response) {
     mocks = kctx.context.mocks;
   }
 
-  keploy.capture({
-    Captured: Date.now(),
-    AppID: keploy.appConfig.name,
-    // change url to uri ex: /url-shortner/:param
-    URI: req.originalUrl,
-    HttpReq: {
-      Method: req.method,
-      URL: req.originalUrl,
-      URLParams: req.params,
-      Header: reqHeader,
-      Body: JSON.stringify(req.body),
-    },
-    HttpResp: {
-      StatusCode: res.statusCode,
-      Header: respHeader,
-      // @ts-ignore
-      Body: String(ResponseBody.get(req)),
-    },
-    Dependency: deps,
-    TestCasePath: keploy.appConfig.testCasePath,
-    MockPath: keploy.appConfig.mockPath,
-    Mocks: mocks,
-    Type: HTTP,
-  });
+  if (kctx.context.mode == "test") {
```

Comment:
Yes, this doesn't seem to work I tested this yesterday on the sample app. I don't know how did it work that time this solution won't record tests in either mode because the context is set to "record" for both cases so this condition won't ever be true. I will change it and commit after testing correctly again.

Reply:
I have made the changes in the next commit, please check them out :smile: thanks

Reply:
LGTM